portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,31 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    String[] quotes = {"You know more than you think you know, just as you know less than you want to know.", 
+                        "One day I will find the right words, and they all will be simple.",
+                        "I assure you that the world is not so amusing as something we imagined."};
+
+    String json = convertToJson(quotes);
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(String[] quotes) {
+    String json = "{";
+    json += "\"quotes\": ";
+    json += "[";
+    for(int i = 0; i < quotes.length; i++) {
```

Comment:
There's actually some helpful utility functions in the GSON library, e.g. https://www.java67.com/2021/12/how-to-convert-array-to-json-in-java.html. I'd recommend exploring the library to see if there's some utilities you can reuse to avoid recreating the wheel :)